src/Process/GlobeTileProcessing.js

```diff
@@ -164,6 +132,7 @@ export function globeSchemeTileWMTS(type) {
 }
 
 export function computeTileZoomFromDistanceCamera(distance, view) {
+    // TODO fixme
```

Comment:
do you have an idea?

Reply:
I didn't even try to fix this for now.

I'd rather make sure that the sse computation introduced in this PR is ok.